### DISCUSSION TITLE
How to fix Laragon slowness

### DISCUSSION THREAD

**Discussion by oabwhmN** [leokhoa/laragon#506] [General]
My Wordpress website was really snappy about a year ago. Now, page loads are slow and Wordpress settings changes are slow to update as well.

Is there any way to speed it up?

**Comment by tvaughan73** [comment]
I am having same issue with a laravel site.  Was really quick a couple of months ago and now it takes up to 10 seconds for a simple request to complete.  Using PHP 8.1 and MySQL 8.

**Reply by DuncanMcCNZ** [reply]
It makes pretty BIG difference to add exceptions to Windows Security for: 
* c:\windows\windows32\conhost.exe
* Your laragon php, apache, and mysql folders.
* Also memcached and redis folders if you use them (and I think you should).

**Reply by hacheraw** [reply]
This worked for me.

The path is ```C:\Windows\System32\conhost.exe``` tho

**Comment by darshan968** [comment]
I am having issue while opening laragon 
this is the windows error I'm getting
Exception EAccessViolation in module laragon.exe at
0015AB79.
Access violation at address 0055AB79 in module 'laragon.exe'.
Read of address 00000B30.
